src/config/__test__/validate.test.ts

```diff
@@ -1,29 +1,62 @@
 import { validateConfig } from '../validate.js'
 import test from 'ava'
 import { RawConfig } from '../types.js'
+import { InvalidUrl } from '../../errors/invalidUrl.js'
+import { config } from '../index.js'
+import { InvalidConfigError } from '../../errors/invalidConfig.js'
 
-const config: RawConfig = {
-	port: 'mock-port',
-	appName: 'mini-test',
-	baseRedirectUrl: 'http://mock.com',
-	url: {
-		matchPattern: 'mock-patter',
-		lifetime: '120',
-	},
-	storage: {
-		driverName: 'InMemory',
-		relationalDriverConfig: {
-			client: 'mock-client',
-			connection: {
-				host: 'local-host',
-				user: 'snir-hamair',
-				password: 'admin1234',
-				database: 'mock-db',
+function getConfig(): RawConfig {
+	return {
+		port: 'mock-port',
+		appName: 'mini-test',
+		baseRedirectUrl: 'http://mock.com',
+		url: {
+			matchPattern: 'mock-patter',
+			lifetime: '120',
+		},
+		storage: {
+			driverName: 'InMemory',
+			relationalDriverConfig: {
+				client: 'mock-client',
+				connection: {
+					host: 'local-host',
+					user: 'snir-hamair',
+					password: 'admin1234',
+					database: 'mock-db',
+				},
 			},
 		},
-	},
+	}
 }
-
 test('validate config - happy flow [configuration set propely]', (t) => {
+	t.is(validateConfig(getConfig()), true)
+})
+
+test('Making sure that validateConfig throws if an invalid baseRedirectUrl is given', (t) => {
```

Comment:
Can you merge all baseRedirectUrl tests into one block (preferably using the array forEach, as the first test)?

The test description should also include the function name which it tests. You can also shorten the description by removing "Making sure" ->'validateConfig throws if an invalid baseRedirectUrl is given'.

Reply:
@SnirShechter Thanks! I'll make sure to merge the tests into one block, shorten the description and add the name of the function which is tested. Furthermore I'll make the pathname test dynamic.